mne/epochs.py

```diff
@@ -1700,7 +1708,7 @@ class EpochsArray(Epochs):
     @verbose
     def __init__(self, data, info, events, tmin=0, event_id=None,
                  reject=None, flat=None, reject_tmin=None,
-                 reject_tmax=None, verbose=None):
+                 reject_tmax=None, baseline=(None, 0), verbose=None):
```

Comment:
Yeah, I think so. The only one I've monitored is that `verbose` should be last, I guess `preload` as second-to-last is fine by me. But it's too late to change it for other functions probably. Just make sure you make the default here `baseline=None` so that it doesn't break people's code.